config.yaml

```diff
@@ -2,18 +2,31 @@
 # See LICENSE file for licensing details.
 
 options:
-  git-sync::repo:
+  git_repo:
     type: string
-    description: URL to repo to sync against
-  git-sync::branch:
+    description: URL to repo to clone and sync against.
+  git_branch:
     type: string
-    description: branch to sync
-    default: main
-  git-sync::wait:
+    description: The git branch to check out.
+    default: master
```

Comment:
Wait why do we need for a branch and a reference? Branches _are_ references in git

Reply:
See reply above.